wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/__rootArtifactId__-web/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="
+      http://xmlns.jcp.org/xml/ns/javaee
+      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
```

Comment:
As WF23 adresses Jakarata EE 8, this should be [https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#declaring_selected_alternatives](https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#declaring_selected_alternatives)
`xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_2_0.xsd" bean-discovery-mode="annotated" version="2.0"`

Reply:
Changed. Thanks for spotting this!